frontend/src/components/CoronaActivitiesComponents/ProvinceCapitalWeather.js

```diff
@@ -43,6 +43,25 @@ const WeatherWrapper = styled.section`
   }
 `
 
+const Headline = styled.section`
+  text-align: center;
+  p + p {
+  margin-top: 0px;
+  }
+`
+
+const CurrentWeather = styled.section`
+  display: flex;
+  flex-flow: row;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  
+  p {
+    font-size: 24px;
```

Comment:
maybe use a relative unit?

Reply:
have that planned in a potential refactoring 🤞🏻  will see how far I can get this weekend 🤣